gcc/doc/invoke.texi

```diff
@@ -21244,14 +21244,25 @@ The following extension options are common to the listed CPUs:
 
 @table @samp
 @item +nodsp
-Disable the DSP instructions on @samp{cortex-m33}, @samp{cortex-m35p}.
+Disable the DSP instructions on @samp{cortex-m33}, @samp{cortex-m35p} and @samp{cortex-m85}. Also disable the M-Profile Vector Extension (MVE) integer and single precision floating-point instructions on @samp{cortex-m85}.
+
+@item +nopacbti
+Disable the Pointer Authentication and Branch Target Identification Extension
+on @samp{cortex-m85}.
```

Comment:
`nopacbti`, which is not a valid option in this version of GCC, should not be mentioned in the man pages.

Reply:
@stephanosio , thank you for your comment. I agree with you. I will update for commit "arm: Remove pacbti in cortex-m85" to remote pacbti information in man pages.

Reply:
@stephanosio , I updated in https://github.com/zephyrproject-rtos/gcc/pull/28/commits/d1b1cc57623f51cd53ca01bebdb2cc075aaac505, please help me review for it. Thank you.